packages/dids/src/index.js

```diff
@@ -0,0 +1,61 @@
+import {Ed25519VerificationKey2020} from '@digitalbazaar/ed25519-verification-key-2020';
+import {CryptoLD} from 'crypto-ld';
+const didKeyDriver = require('@digitalbazaar/did-method-key').driver();
+
+export const DIDKeyManager = (function () {
+  let wallet;
+
+  const createDidKeyPair = async () => {
+    const cryptoLd = new CryptoLD();
+    cryptoLd.use(Ed25519VerificationKey2020);
+    return cryptoLd.generate({type: 'Ed25519VerificationKey2020'});
+  };
+  const saveDiDKeyPair = keyPair => {
+    return saveToWallet({
+      type: 'KEY',
```

Comment:
this class shouldnt be saving keypairs, it should ideally take a key document from the wallet to produce the DID